pkg/ocm/policylist.go

```diff
@@ -0,0 +1,42 @@
+package ocm
+
+import (
+	"context"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	policiesv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// ListPoliciesInAllNamespaces returns a cluster-wide policy inventory.
+func ListPoliciesInAllNamespaces(apiClient *clients.Settings,
+	options runtimeclient.ListOptions) (
```

Comment:
One last thing, we actually just made an update so that the options param for list functions is optional. Can you align this function with this example - https://github.com/openshift-kni/eco-goinfra/blob/main/pkg/bmh/list.go#L29-L41